frontend/src/Context/SessionProvider/session.tsx

```diff
@@ -10,50 +10,34 @@ import {
   useEffect,
   ReactElement,
 } from 'react';
-import { v4 as uuid } from 'uuid';
-import {
-  Connection,
-  Dimensions,
-  Session,
-  Stream,
-  Subscriber,
-  SubscriberProperties,
-  initSession,
-  Event,
-  OTError,
-} from '@vonage/client-sdk-video';
+import { Connection, Publisher, Stream } from '@vonage/client-sdk-video';
 import fetchCredentials from '../../api/fetchCredentials';
-import createMovingAvgAudioLevelTracker from '../../utils/movingAverageAudioLevelTracker';
 import useUserContext from '../../hooks/useUserContext';
 import ActiveSpeakerTracker from '../../utils/ActiveSpeakerTracker';
 import useRightPanel, { RightPanelActiveTab } from '../../hooks/useRightPanel';
-import logOnConnect from '../../utils/logOnConnect';
+import {
+  Credential,
+  SignalEvent,
+  SubscriberAudioLevelUpdatedEvent,
+  SubscriberWrapper,
+} from '../../types/session';
 import useChat from '../../hooks/useChat';
-import { SubscriberWrapper } from '../../types/session';
 import { ChatMessageType } from '../../types/chat';
 import { isMobile } from '../../utils/util';
 import {
   sortByDisplayPriority,
   togglePinAndSortByDisplayOrder,
 } from '../../utils/sessionStateOperations';
 import { MAX_PIN_COUNT_DESKTOP, MAX_PIN_COUNT_MOBILE } from '../../utils/constants';
+import VonageVideoClient from '../../utils/VonageVideoClient';
+import useEmoji, { EmojiWrapper } from '../../hooks/useEmoji';
 
 export type { ChatMessageType } from '../../types/chat';
 
-export type ChangedStreamType = {
```

Comment:
This and a few other types were moved to a separate file so the `VonageVideoClient` class would not have a circular dependency